src/openforms/config/constants.py

```diff
@@ -85,7 +85,10 @@ class UploadFileType(models.TextChoices):
         "application/vnd.oasis.opendocument.*,application/vnd.stardivision.*,application/vnd.sun.xml.*",
         _("Open Office"),
     )
-    zip = "application/zip", _(".zip")
+    zip = (
+        "application/zip," "application/zip-compressed," "application/x-zip-compressed",
```

Comment:
```suggestion
        "application/zip,application/zip-compressed,application/x-zip-compressed",
```
unless this was deliberate, but in that case I don't understand why it was written that way :P